portfolio/src/main/webapp/servletScript.js

```diff
@@ -0,0 +1,13 @@
+/*
+* Calls fetch() and adds response to the webpage
+*/
+
+async function fetchString(){
+    const responseFromServer = await fetch('/funny');
+    const jsonFromServ = await responseFromServer.json();
+
+    console.log(jsonFromServ);
+
+    const linuxParagraph = document.getElementById("linux");
+    linuxParagraph.innerText = jsonFromServ[Math.floor(Math.random() * jsonFromServ.length)];
```

Comment:
I don't recall the exact requirements of this week's task but you could potentially have a /funny/random servlet API that just picks a random joke and returns just a single string and does the randomization server-side.

No need to change anything, it's just useful to think about where you want logic to live in client-server applications. Neither option is more correct than the other in this case.

Side note: Why is the paragraphs ID "linux"? :)

Reply:
Yeah that's a good point, would it be faster that way? Though I think the task wanted us to do it this way. I forgot to change the IDs and I actually fixed it after I sent the commit